src/components/ExportButton/ExportButton.js

```diff
@@ -0,0 +1,194 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { jsPDF as JsPDF } from 'jspdf';
+import { useTranslation } from 'react-i18next';
+import { parse as parseSvg, stringify as stringifySvgObject } from 'svgson';
+import { makeStyles } from '@material-ui/core';
+import CanvasSaveButton from 'react-spatial/components/CanvasSaveButton';
+import Canvg from 'canvg';
+import { ReactComponent as Loader } from './loader.svg';
+import legend from './tarifverbund_legend.svg';
```

Comment:
Can we make this a bit more generic to support other topics?

Reply:
I moved the legend data url to the topic exportConfig, so in the future we can handle exports on topic level